server/form-pages/apply/risks-and-needs/health-needs/physicalHealth.ts

```diff
@@ -97,6 +97,14 @@ export default class PhysicalHealth implements TaskListPage {
       errors.hasPhyHealthNeeds = 'Confirm whether they have physical health needs'
     }
 
+    if (this.body.hasPhyHealthNeeds === 'yes' && !this.body.needsDetail) {
```

Comment:
very minor but the two cases could be brought withing a single if statement, which might more intuitively reflect the data structure here?

i.e. 
```
if (this.body.hasPhyHealthNeeds === 'yes') {
      if (!this.body.needsDetail) {
            errors.needsDetail = 'Describe physical health needs'
     }
      if (!this.body.canClimbStairs) {
            errors.canClimbStairs = 'Confirm whether they can climb stairs'
     }
  }
```

Reply:
thks, yes that's clearer: I'll tweak that